pom.xml

```diff
@@ -95,38 +87,6 @@
                 <artifactId>maven-war-plugin</artifactId>
                 <version>2.4</version>
             </plugin>
-            <plugin>
-                <groupId>com.mycila</groupId>
-                <artifactId>license-maven-plugin</artifactId>
-                <version>3.0</version>
-                <configuration>
-                    <header>com/mycila/maven/plugin/license/templates/AGPL-3.txt</header>
-                    <mapping>
-                        <java>SLASHSTAR_STYLE</java>
-                    </mapping>
-                    <properties>
-                        <owner>Andrew Rice</owner>
```

Comment:
The issue I had with this plugin is I couldn't work out how to make it accept different licence authors or years for different files. If there is a way, that would be fine, but as it is, I don't think that would work as some files are just `Andrew Rice`, and some copyright years need updating.

Reply:
The faq describes an options for executing the plugin twice: https://www.mojohaus.org/license-maven-plugin/faq.html - how about that?

Reply:
In the check licence mode, we could run the plugin twice, but I think it would need a list of files that are exclusively copyright to you for one execution, and a list where the copyright is shared for the other. That seems fiddly to maintain, especially as the number of contributors increases. What do you think?

Reply:
Yeah. That's pretty horrible. Let's just tag all the files Andrew Rice and BlueOptima and be done with it?